tests/HomePage/HomePage.logic.test.js

```diff
@@ -0,0 +1,65 @@
+import { getArticles } from "../../src/pages/HomePage/services";
+import * as fetch from "node-fetch";
```

Comment:
In tests you shouldn't use the real API. Please mock `fetch`. E.g., you can use `jest-fetch-mock` library for that